Capstone/Autocorrect/AutocorrectFunctions.m

```diff
@@ -43,45 +62,46 @@ - (void)createCoordinates {
     [self.keyCoordinates setObject:@"65" forKey:@"m"];
 }
 
-- (double)hammingDistance:(NSString *)string1 :(NSString *)string2 {
+- (double)hammingDistanceCalculator:(NSString *)string1 :(NSString *)string2 {
+    [self createCoordinates];
     double sum = 0;
     int l1 = string1.length;
     int l2 = string2.length;
```

Comment:
Let's define char 1 and 2 up here so you don't have to keep redefining it in the nested if statements
```suggestion
NSString *character1;
NSString *character2;

Reply:
Done!